openmrs/api/src/main/java/org/projectbuendia/openmrs/api/db/hibernate/HibernateProjectBuendiaDAO.java

```diff
@@ -31,4 +37,21 @@ public SessionFactory getSessionFactory() {
     public void setSessionFactory(SessionFactory sessionFactory) {
         this.sessionFactory = sessionFactory;
     }
+
+    @Override
+    public List<Encounter> getEncountersCreatedAtOrAfter(Date date) {
+        // NOTES:
+        // - this code relies on the assumption that observations can't be modified independently of
+        // encounters.
+        // - This doesn't actually return encounters "modified on or after", it's currently
+        // encounters "created on or after". This is ok for our purposes because we don't have
+        // the ability to modify Encounters in Buendia.
```

Comment:
I agree that should be fine. Encounters or observations should never be modified. To correct an observation, we would void the old observation and add a new observation with the new value.